tests/Support/Traits/CssDataProviders.php

```diff
@@ -29,19 +29,19 @@ public function provideEquivalentCompleteCss(): array
     {
         $datasetsWithAtMediaRuleSelectorListAndPropertyDeclaration = self::crossDatasetWithItself([
             'unminified CSS with `@media` rule, selector list, and property declaration'
-                => ['@media screen { html, body { color: green; } }'],
+            => ['@media screen { html, body { color: green; } }'],
```

Comment:
I think this is more readable with the indentation.

We would indent a long variable assignment, and not write

```php
$longVariableName
= $this->longExpression($someValue);
```

I think defining an array value is the same kind of thing.